binding.cc

```diff
@@ -1773,21 +1773,30 @@ NAPI_METHOD(batch_clear) {
  */
 struct BatchWriteWorker final : public PriorityWorker {
   BatchWriteWorker (napi_env env,
+                    napi_value thisObj,
```

Comment:
Maybe I should I rename this to "context" or "batchObj". Other suggestions?

Reply:
`batchObj` sounds a little confusing alongside with `*batch`.

Reply:
Agree. What about "context"? That would match up with the name used in JS land:

https://github.com/Level/leveldown/blob/7312cbbe01c6191eaca8e8df7b99395211635585/chained-batch.js#L7

Reply:
👍